build/kube_setup.sh

```diff
@@ -252,7 +252,8 @@ function install_elasticsearch_operator {
     fi
 
     echo "*** Applying elastic-operator manifest ..."
-    ${KUBECTL_CMD} apply -f https://download.elastic.co/downloads/eck/1.6.0/all-in-one.yaml
+    ${KUBECTL_CMD} create -f https://download.elastic.co/downloads/eck/1.7.1/crds.yaml
```

Comment:
Any reason for using `create` vs `apply`?

Reply:
Pulled instructions from https://www.elastic.co/downloads/elastic-cloud-kubernetes